test/acceptance/features/step_definitions/common.js

```
@@ -0,0 +1,9 @@
+const { get } = require('lodash')
+const { client } = require('nightwatch-cucumber')
+const { When } = require('cucumber')
+
+When(/^I navigate to the "(.+)" page$/, async function (pageName) {
+  const Page = get(client.page, pageName)()
```

So your always relying on getting a function here, which means it will blow up when the `get` returns `undefined`. Maybe separate out the page object function call into an `isFunction` check?

---

I'm in two minds. I think it's good to maybe protect against that but we should still throw an error?

If you call this step definition with a bad value we want it to fail I suppose, not to silently carry on?

---

Not sure if there is a need to throw an error. Won't it fail because it doesn't navigate to the expected page? Rather than failing because it blows up?